packages/website/src/components/page-templates/documents-index.tsx

```diff
@@ -37,33 +48,33 @@ class DocsTable extends React.Component {
           </Headers>
           <Rows
             items={pageInfo[docKey]}
-            render={({ id, pagePath, children }) => (
+            render={(page: DocsPage) => (
```

Comment:
What's the reason for de-deconstructing this?

Reply:
As discussed, I needed to pass the full page object to `getTitle` so just skipped the destructure rather than converting the arrow function to a block and deconstructing on the next line.